socorro/external/fs/crashstorage.py

```diff
@@ -416,10 +416,8 @@ def save_raw_crash(self, raw_crash, dumps, crash_id):
             try:
                 self._create_date_to_name_symlink(crash_id, slot)
                 self._create_name_to_date_symlink(crash_id, slot)
-            except OSError:
-                # We pass here, because an OSError indicates the symlink
-                # already exists and we don't need to do anything about that.
-                pass
+            except OSError as exc:
+                self.logger.info('failed to create symlink: %s', str(exc))
```

Comment:
Why not 
```python
self.logger.info('failed to create symlink', exc_info=True)
```
?
Also, why not `self.logger.warning` (or whatever it's called). Or even, if the exception is worth seeing, why not `self.logger.error`?